packages/react/src/alert/Alert.tsx

```diff
@@ -0,0 +1,68 @@
+import { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { Button } from "../button";
+import { Flex } from "../flex";
+import { IconDanger } from "../icons/IconDanger";
+import { IconInfoCircle } from "../icons/IconInfoCircle";
+import { IconSuccess } from "../icons/IconSuccess";
+import { IconWarning } from "../icons/IconWarning";
+import { IconX } from "../icons/IconX";
+import * as styles from "./Alert.css";
+
+type AlertProps = BoxProps<
+  "div",
+  {
+    onClose?: () => void;
+  } & styles.AlertVariants
+>;
+
+const iconMap = new Map([
+  ["danger", IconDanger],
+  ["info", IconInfoCircle],
+  ["success", IconSuccess],
+  ["warning", IconWarning],
+]);
+
+const getIcon = (type: string) => {
+  const IconComponent = iconMap.get(type);
+  return IconComponent ? <IconComponent /> : null;
+};
+
+export const Alert = forwardRef<HTMLDivElement, AlertProps>(
+  (
+    {
+      children,
+      onClose,
+      size = "md",
+      title,
+      type = "info",
+      variant = "light",
+      ...props
+    },
+    ref,
+  ) => {
+    return (
+      <Flex ref={ref} {...styles.alert({ size, type, variant })} {...props}>
```

Comment:
need to handle className here
```suggestion
      <Flex ref={ref} {...styles.alert({ size, type, variant }, className)} {...props}>
```